tools/plasmid_profiler/plasmid_profiler.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="plasmid_profiler" name="Plasmid Profiler" version="0.1.1">
+<tool id="plasmid_profiler" name="Plasmid Profiler" version="0.1.3">
```

Comment:
Is the tool wrapper following the same version number as the tool itself? Wrapper version is 0.1.3 and the tool is 0.1.4